src/NUnitFramework/tests/Constraints/ThrowsExceptionConstraintTests.cs

```diff
@@ -45,6 +46,7 @@ public void SetUp()
         [Test]
         public void SucceedsWithNonVoidReturningFunction()
         {
+
```

Comment:
I'm sorry I didn't notice this earlier. I don't think we should add this blank line.

Reply:
Oops, no, that's my fault. I accidentally added that just now. Thanks for catching, removing now.